bin/sozo/src/commands/options/account.rs

```diff
@@ -42,14 +43,18 @@ impl AccountOptions {
     where
         P: Provider + Send + Sync,
     {
+        trace!(account_options=?self, "Creating account.");
         let account_address = self.account_address(env_metadata)?;
+        trace!(?account_address, "Account address determined.");
+
         let signer = self.signer.signer(env_metadata, false)?;
+        trace!(?signer, "Signer obtained.");
 
         let chain_id =
             provider.chain_id().await.with_context(|| "Failed to retrieve network chain id.")?;
-
+        trace!(?chain_id, "Chain ID obtained.");
```

Comment:
```suggestion
        trace!(?chain_id);
```
To be consistent with the migration output.